content/en/seachat/getting-started/02-quick-start.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Quickstart"
+description: "SeaChat Intro and getting started"
+lead: ""
+date: 2024-03-08 10:43:51.069 +0100
+lastmod: 2024-03-08 10:43:51.069 +0100
+weight: 202
+draft: false
+images: []
+aliases:
+  - /en/seachat/seachat-intro/02-quick-start/
+  - /seachat/seachat-intro/02-quick-start/
+---
+---
+* In just under 10 minutes, you'll have your own customized AI agent set up and ready to deploy on your website.
+* To get a quick taste of SeaChat, you can simply log into your SeaChat account now and follow our hands-on video tutorial to start.
```

Comment:
don't direct the user to another page and watch another video. just delete 2nd bullet point